envoy/access_log/access_log.h

```diff
@@ -14,6 +15,8 @@
 namespace Envoy {
 namespace AccessLog {
 
+using HttpLogContext = Formatter::HttpFormatterContext;
```

Comment:
Instead of aliasing this class should we move it somewhere more general purpose? (Or just use the full name so we don't have 2 different names for the same class)

Reply:
sgtm.

Reply:
Hmmm, it's OK to use `Formatter::HttpFormatterContext` for formatter-based loggers. But it seems weird to use this name for logger like grpc.

Reply:
Let's use full name first. I don't want to introduce the name change in this patch.